Hippo.Core/Hippo.Core.csproj

```diff
@@ -20,7 +20,6 @@
 	<ItemGroup>
 		<Folder Include="Migrations\SqlServer\" />
 		<Folder Include="Migrations\Sqlite\" />
-		<Folder Include="Utilities\" />

```

Comment:
Don't know why this was removed.... Maybe because it was an empty folder before?

Reply:
don't know but you shouldn't need it